student-bios/MadhuMNG.md

```diff
@@ -0,0 +1,33 @@
+# Hello everyone
+
+I am Madhu.I am basically from India. Since 5 years I am staying in Belgium.
+
+## Linguistic capabilities
+
+1. Kannada
+2. English
+3. French and Dutch
+
+### Education
+
+- I have done Master in Business Adninistration from Bangalore University.
+- I am currenting learning French-B1 level.
+- I am a student in HYF.
+
+## Work Experience
+
+| Organisation | Role                                             |
+| ------------ | ------------------------------------------------ |
+| Ernst&Young  | Advanced tax analyst                             |
+| TCS          | Project managing officer , Subject matter expert |
+
+### Hobbies
+
+:joy: I am passion about Artistic activities such as painting, pencil sketch,
+crochet, Resin jewellery making
+
+(my insta page) <https://www.instagram.com/padma_vibhooshita >.
```

Comment:
@MadhuMNG on line 29, you should use [ ] for the text part, and ( ) for the url part. Just try it out!

- And before pushing changes to github, it might be a good to make sure that it is viewed correctly with the preview feature of VS Code